.gitignore

```diff
@@ -0,0 +1,2 @@
+__pycache__
+.DS_Store
```

Comment:
Can't say I've ever heard of that gitignore philosophy, but it's your project. :)

Reply:
Thanks, this is just because everyone has a different dev environment, that may create different sets of temporary files. So a single gitignore file isn't going to catch all cases, that's why I believe people should setup their environments to their liking and learn how to manage their gitignore file. 😄